core/src/main/java/org/apache/calcite/rel/RelDistributions.java

```diff
@@ -81,6 +77,15 @@ public static RelDistribution of(RelDistribution.Type type, ImmutableIntList key
     return RelDistributionTraitDef.INSTANCE.canonize(distribution);
   }
 
+  public static ImmutableIntList normalizeKeys(Collection<? extends Number> numbers) {
```

Comment:
Can you change it to private and add a comment?

Reply:
Done.